src/main/java/hudson/plugins/throttleconcurrents/ThrottleJobProperty.java

```diff
@@ -66,7 +73,20 @@ public ThrottleJobProperty(Integer maxConcurrentPerNode,
                 new CopyOnWriteArrayList<String>(categories);
         this.throttleEnabled = throttleEnabled;
         this.throttleOption = throttleOption;
+        this.limitOneJobWithMatchingParams = limitOneJobWithMatchingParams;
         this.matrixOptions = matrixOptions;
+        this.paramsToUseForLimit = paramsToUseForLimit;
+        if ((this.paramsToUseForLimit != null)) {
+            if ((this.paramsToUseForLimit.length() > 0)) {
+                this.paramsToCompare = Arrays.asList(this.paramsToUseForLimit.split(","));
+            }
+            else {
+                this.paramsToCompare = new ArrayList<String>();
+            }
+        }
+        else {
+            this.paramsToCompare = new ArrayList<String>();
+        }
```

Comment:
`StringUtils.split` provides `split` + null check.
`ArrayUtils.nullToEmpty` converts null to empty array.
They will help you.